.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/labeler#common-examples
+# Generated from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cudf/labels
+
+cuDF (Python):
+  - 'python/**'
+  - 'notebooks/**'
+  
+libcudf:
+  - 'cpp/**'
+
+CMake:
+  - '**/CMakeLists.txt'
+  - '**/cmake/**'
+  
+cuDF (Java):
+  - 'java/**'
+  
+Ops:
+  - '.github/**'
+  - 'ci/**'
```

Comment:
We also have a `gpuCI` label, but not sure if it makes sense to have that for this.

Reply:
Would be helpful to call this out automatically, we should create a label section just for the `ci` path